ssh/tailssh/tailssh.go

```diff
@@ -48,8 +48,19 @@ var (
 	sshVerboseLogging = envknob.RegisterBool("TS_DEBUG_SSH_VLOG")
 )
 
+// LocalBackend is the subset of ipnlocal.LocalBackend that we use.
+// It is used for testing.
+type LocalBackend interface {
```

Comment:
does it need to be exported? could call it ipnLocalBackend if you want to keep LocalBackend part capitalized.